pkg/R/types.R

```diff
@@ -0,0 +1,47 @@
+# Utility functions for handling Spark DataTypes.
```

Comment:
I'd probably change this to SparkSQL types as I am not sure what other types we'll need to do this for.